src/main/java/Player.java

```diff
@@ -1,20 +1,25 @@
 import java.util.ArrayList;
 import java.util.Random;
 
-public class Player {
+public class Player
+{
     public ArrayList<Character> team =  new ArrayList<>(4);
-    public void createTeam() {
-        CharacterList characterList1  = new CharacterList();
-        CharacterList characterList2  = new CharacterList();
-        CharacterList characterList3  = new CharacterList();
-        team.add(characterList1.pH);
-        characterList1.createCharList();
-        characterList2.createCharList();
-        characterList3.createCharList();
-        team.add(characterList1.charList.get(new Random().nextInt(characterList1.charList.size())));
-        team.add(characterList2.charList.get(new Random().nextInt(characterList2.charList.size())));
-        team.add(characterList3.charList.get(new Random().nextInt(characterList3.charList.size())));
-    }
 
+    public void createTeam()
+    {
+        CharacterList characterList  = new CharacterList();
+
+        team.add(characterList.pH);
+        characterList.charList = characterList.createCharList();
 
+        team.add(characterList.charList.get(new Random().nextInt(characterList.charList.size())));
+        team.add(new CharacterList().createCharList().get(new Random().nextInt(characterList.charList.size())));
+        team.add(new CharacterList().createCharList().get(new Random().nextInt(characterList.charList.size())));
```

Comment:
I added this logic back because apparently if a team is created with one list, if there's duplicates in the team, it treats them the same in-game. Ex. you have 2 Ogres on your team, and only one of them takes damage. when it shows you the health again, it shows that they both took damage. My best guess as to why this happens is since there's a copy of the same character in memory, it updates both when one is updated. 🤷‍♂️